cmd/incus/network_zone.go

```diff
@@ -122,15 +124,25 @@ func (c *cmdNetworkZoneList) Run(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf(i18n.G("Filtering isn't supported yet"))
 	}
 
-	zones, err := resource.server.GetNetworkZones()
-	if err != nil {
-		return err
-	}
+    var zones []api.NetworkZone
+    if c.flagAllProjects {
+        zones, err = resource.server.GetNetworkZonesAllProjects()
+        if err != nil {
+            return err
+        }
+    } else {
+        zones, err = resource.server.GetNetworkZones()
+        if err != nil {
+            return err
+        }
+    }
+
 
 	data := [][]string{}
 	for _, zone := range zones {
 		strUsedBy := fmt.Sprintf("%d", len(zone.UsedBy))
 		details := []string{
+            zone.Project, 
```

Comment:
Similar to what we've done elsewhere, the extra Project column should only be showed if `flagAllProjects` is true